app/assets/stylesheets/dashboard/service_calendar.sass

```diff
@@ -1,3 +1,14 @@
+.user-view-ssr-modal
+  width: 95%
+
+.visit-before-container, .visit-day-container, .visit-after-container
```

Comment:
I think we should leave the class on the elements. It's a more verbose system than assigning it in the CSS. Thoughts?